components/job-orchestration/job_orchestration/executor/compress/fs_compression_task.py

```diff
@@ -270,10 +317,9 @@ def run_clp(
     # Close stderr log file
     stderr_log_file.close()
 
-    worker_output = {
-        "total_uncompressed_size": total_uncompressed_size,
-        "total_compressed_size": total_compressed_size,
-    }
+    if s3_write_failed:
```

Comment:
Maybe just use `compression_successful`? Or if you want to distinguish it from the compression operation itself, maybe rename `compression_successful` to `operation_successful`?

Reply:
Should be resolved.